### DISCUSSION TITLE
How to apply linode.firewall rules to linode.lkeclusters instead of linode.instance

### DISCUSSION THREAD

**Discussion by EzekielHub** [pulumi/pulumi#12468] [General]
Example i have this domain record connected to a nginx ingres controller on a linode kubernetes cluster. But i want to add some firewall rules to the webiste. how can i connect the firewall rule to the domain record.
thanks in advance

```python
foobar_domain_record = linode.DomainRecord("foobarDomainRecord",
    domain_id=foobar_domain.id,
    name="www",
    record_type="CNAME",
    target="foobar.example")


my_firewall = linode.Firewall("myFirewall",
    label="my_firewall",
    inbounds=[
        linode.FirewallInboundArgs(
            label="allow-http",
            action="ACCEPT",
            protocol="TCP",
            ports="80",
            ipv4s=["0.0.0.0/0"],
            ipv6s=["::/0"],
        ),
        linode.FirewallInboundArgs(
            label="allow-https",
            action="ACCEPT",
            protocol="TCP",
            ports="443",
            ipv4s=["0.0.0.0/0"],
            ipv6s=["::/0"],
        ),
    ],
    inbound_policy="DROP",
    linodes=[my_instance.id])
```

**Comment by t0yv0** [comment]
What's your high-level goal, are you trying to have traffic that comes to your domain, like GET http://foobar.example be served by your Linode instance? Firewall connects to the instance as your code does already. Perhaps look at adding a https://www.pulumi.com/registry/packages/linode/api-docs/instanceip/  resource that would generate a public IP address for your Linode instance, and then create a domain record that maps a DNS name to this public IP address?

BTW if you think there's an issue with pulumi please open an issue in https://github.com/pulumi/pulumi-linode this will help my team to triage it.
